blacklight-locale_picker.gemspec

```diff
@@ -13,16 +13,16 @@ Gem::Specification.new do |spec|
   spec.summary     = ""
   spec.description = ""
   spec.license     = "Apache-2.0"
-  
+
   spec.files         = `git ls-files -z`.split("\x0").reject do |f|
     f.match(%r{^(test|spec|features)/})
   end
   spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 2.5.0' 
-  spec.add_dependency "rails", "~> 5.2.3"
+  spec.required_ruby_version = '>= 2.5.0'
+  spec.add_dependency "rails", "< 6.1"
```

Comment:
```suggestion
  spec.add_dependency "rails", ">= 5.2.3", "< 6.1"
```

🤷

Reply:
5.2.3 was problematic with a build on some app, that I can't remember now, but as long as it works, 👍